.github/workflows/scripts/ComposerChecks.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+check_composer_install_contents() {
+  local string_to_check="$1"
+  local expected="$2"
+  local file="composer-install-output.txt"
+
+  ddev composer install > $file 2>&1
```

Comment:
Let's move all of these tests to the composer-checks repository. They don't need to be drainpipe specific. In fact, it would be reasonable to simplify this and use github's setup-php action to test this without ddev.

Some? All? of this may be better as a phpunit test too. Here's an example: https://github.com/deviantintegral/composer-gavel/blob/main/tests/ComposerVersionRequirementTest.php